sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -263,9 +250,9 @@ export interface ReceiveMessagesOptions extends OperationOptionsBase {
 export type ReceiveMode = "peekLock" | "receiveAndDelete";
 
 // @public
-export interface Response {
+export type Response<T> = T & {
     _response: HttpOperationResponse;
```

Comment:
shouldn't this be `HttpResponse`  instead? `HttpOperationResponse` has a lot of internal details that don't add much value to the consumer, whereas `HttpResponse` is enough to get headers, status, and the original request.

Reply:
Hmmm.. we never cared enough to update. We can make it HTTPResponse.

For reference,
```ts
// Service Bus
_response: HttpOperationResponse;

// From core-http
export interface HttpOperationResponse extends HttpResponse {
  parsedHeaders?: { [key: string]: any };
  bodyAsText?: string | null;
  parsedBody?: any;
  blobBody?: Promise<Blob>;
  readableStreamBody?: NodeJS.ReadableStream;
}

// Example for `_response` in storage 
_response: HttpResponse & {
  parsedHeaders: ServiceGetUserDelegationKeyHeaders;
  bodyAsText: string;
  parsedBody: UserDelegationKeyModel;
};
```

Reply:
Since this is beyond the scope of this PR, logging a new issue to update to `_response: HTTPResponse` - #10841
I'll put up a PR for this if nobody has objections.

Reply:
`Response` has been updated to `WithResponse`.